src/graph/UserAccessControl.cpp

```diff
@@ -0,0 +1,194 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "UserAccessControl.h"
+
+namespace nebula {
+namespace graph {
+
+#define    ACL_GO                 (1<<0)
+#define    ACL_SET                (1<<1)
+#define    ACL_PIPE               (1<<2)
+#define    ACL_USE                (1<<3)
+#define    ACL_MATCH              (1<<4)
+#define    ACL_ASSIGNMENT         (1<<5)
+#define    ACL_CREATETAG          (1<<6)
+#define    ACL_ALTERTAG           (1<<7)
+#define    ACL_CREATEEDGE         (1<<8)
+#define    ACL_ALTEREDGE          (1<<9)
+#define    ACL_DESCRIBETAG        (1<<10)
+#define    ACL_DESCRIBEEDGE       (1<<11)
+#define    ACL_DROPTAG            (1<<12)
+#define    ACL_DROPEDGE           (1<<13)
+#define    ACL_INSERTVERTEX       (1<<14)
+#define    ACL_INSERTEDGE         (1<<15)
+#define    ACL_SHOW               (1<<16)
+#define    ACL_DELETEVERTEX       (1<<17)
+#define    ACL_DELETEEDGE         (1<<18)
+#define    ACL_FIND               (1<<19)
+#define    ACL_ADDHOSTS           (1<<20)
+#define    ACL_REMOVEHOSTS        (1<<21)
+#define    ACL_CREATESPACE        (1<<22)
+#define    ACL_DROPSPACE          (1<<23)
+#define    ACL_DESCRIBESPACE      (1<<24)
+#define    ACL_YIELD              (1<<25)
+#define    ACL_CREATEUSER         (1<<26)
+#define    ACL_DROPUSER           (1<<27)
+#define    ACL_ALTERUSER          (1<<28)
+#define    ACL_GRANT              (1<<29)
+#define    ACL_REVOKE             (1<<30)
+#define    ACL_CHANGEPASSWORD     (1<<31)
```

Comment:
I would recommend to use `enum`s.

Besides, you have used  up all of the 32 bits, while there are definitely new statements in future. So, You should use a 64bits unsigned integer to represent ACLs instead. BTW. `1` ought to be `1UL`.

Reply:
`enum` is better. That makes sense 👍 
32 bits is really not enough if a new operation is added，I will change it to 64 bits.